tests/test_interpolation.py

```diff
@@ -477,3 +714,455 @@ def test_optional_after_interpolation() -> None:
     # Ensure that we can set an optional field to `None` even when it currently
     # points to a non-optional field.
     cfg.opt_num = None
+
+
+def test_empty_stack() -> None:
+    """
+    Check that an empty stack during ANTLR parsing raises a `GrammarParseError`.
+    """
+    with pytest.raises(GrammarParseError):
+        grammar_parser.parse("ab}", lexer_mode="VALUE_MODE")
+
+
+def _maybe_create(definition: str) -> Any:
```

Comment:
anything is using this function?

Reply:
Nope, I didn't realize that coverage didn't run on tests.
Trivially deleted in aa842d8 => resolving